test/helpers/matchers.js

```diff
@@ -111,12 +117,19 @@ beforeEach(function() {
           };
         }
       };
+    },
 
-      function DOMTester(a, b) {
-        if (a && b && a.nodeType > 0 && b.nodeType > 0) {
-          return a === b;
+    toEqualOneOf: function(util) {
+      return {
+        compare: function(actual) {
+          var expectedArgs = Array.prototype.slice.call(arguments, 1);
```

Comment:
It's kind of weird to use internal Angular functions in matchers. IMO tests shouldn't depend on the code they're testing. Also, all other relevant matchers in this file use this technique so changing it just here would be inconsistent.